core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -548,16 +521,9 @@ private[spark] class BlockManager(
    * This does not acquire a lock on this block in this JVM.
    */
   def getRemote(blockId: BlockId): Option[BlockResult] = {
```

Comment:
can you rename this `getRemoteValues` to be consistent? Now we have
```
getLocalValues
getLocalBytes
getRemote
getRemoteBytes
```
so this one stands out